frontend/src/routes/Governance/policies/policy-details/PolicyDetailsResults.tsx

```diff
@@ -92,7 +92,7 @@ export default function PolicyDetailsResults() {
             kind: template?.objectDefinition.kind ?? '-',
             status: detail.compliant ?? 'no-status',
             message: (detail?.history && detail.history[0]?.message) ?? '-',
-            timestamp: detail?.history && detail?.history[0]?.lastTimestamp,
+            timestamp: detail?.history?.[0]?.lastTimestamp ?? new Date().toISOString(),
```

Comment:
@JustinKuli  @mprahl  If` detail?.history?.[0]?.lastTimestamp` is undefined or null, Should we return '-' like other values?

Reply:
That sounds good to me.

Reply:
just to confirm, @mprahl @yiraeChristineKim  it is yes for `'-'` when  `detail?.history?.[0]?.lastTimestamp` returns null or undefined